tests/test_parse_load_args.py

```diff
@@ -0,0 +1,68 @@
+import argparse
+import unittest
+import uuid
+from typing import List, Optional
+
+import mkosi
+import mkosi.backend as backend
+from mkosi.backend import MkosiArgs, MkosiException, Distribution
+
+
+def load_args_wrapper(argv: Optional[List[str]] = None) -> MkosiArgs:
+    return mkosi.load_args(mkosi.parse_args(argv)["default"])
+
+
+class TestMkosiMethods(unittest.TestCase):
+
+    def test_parse_load_verb(self):
+        self.assertIsInstance(load_args_wrapper([]), MkosiArgs)
```

Comment:
This one is superfluous and can be removed. We already have typing to verify load_args_wrapper returns MkosiArgs.